specs/testing/skills/SkillsFunctionalTesting.md

```diff
@@ -128,16 +129,42 @@ Given these elements, the number of test cases for each scenario can be calculat
 **Alternate flows**
 
 1. The skill creates a conversation (in teams) and starts a 1:1 conversation with a user in the group. Note - the 1:1 conversations created persist, and there is no way to delete them. Repeated calls to createConversation will succeed however, and return the appropriate conversationId that can be re-used.
+2. **NOTE: Are there meaningful differences in other channels between create and continue conversation?**
 
 **Total test cases:** 96 (not including alternate flows)
 
+### 3a. Skill sends a proactive message to consumer - create conversation
+
+> A consumer calls a _reminder skill_ and the skill finishes the conversation but stores some tasks. Then, some time later the skill creates a new conversation with the consumer and sends a message.
+
+
+**Testing matrix**
+
+- Skill: RemindMe
```

Comment:
Add a link to a placeholder in the Skills section of the doc where we can write the description for this skill.

Reply:
updated.